addons/indicators/functions/fnc_display.sqf

```diff
@@ -35,7 +36,14 @@ private _size = GVAR(size);
             if (_distance > _disanceWarning || {!(isNull objectParent _x)}) then {
                 _color = + (_x getVariable [QEGVAR(radar,compass_color), [1,1,1]]);
             };
-            _alpha = _alpha * (linearConversion [10, _range, _distance, _clamp, 0, true]);
+            if (_useACE) then {
+                // ace_nametags_drawParameters is the parameters of a ACE cachedCall
+                // We do select 1 to get it's parameters, which are the parameters for a `drawIcon3D` call
+                // We then do select 1 to get the color for that call, select 3 is to get the alpha of that color
+                _alpha = (_x getVariable "ace_nametags_drawParameters") select 1 select 1 select 3;
```

Comment:
I'd prefer here to have a `min` check for the original `_alpha` value. Reason is if the player is using occlusion, then the icon is still completely visible as indicator while the compass icon is slowly fading away.

Reply:
Additionally it seems that ace nametags does a LOS check. The names disappear but the icons are still there. That is however an issue if a unit goes behind some object and is obstructed. The alpha value does not change anymore and the indicator stays visible until the original range value from the settings exceeds the distance to that unit.
![image](https://user-images.githubusercontent.com/8259498/54532863-23b62e80-4989-11e9-9604-dadf46c7945f.png)

See here, one obstructed unit, the others are all visible.